### ISSUE TITLE
Error message on all releases starting with 2021.12.2

### ISSUE THREAD

**Issue by meatee** [andresdelcampo/GameList_Editor#18]
I'm new to Gamelist Editor (starting today), and I'm having trouble running the newer versions. I'm getting the error message below (see attached image) with every release starting with 2021.12.2

I can run 2021.12.1 and before just fine.

I'm on Windows 10 Pro

![Screenshot 2024-01-21 135959](https://github.com/andresdelcampo/GameList_Editor/assets/7986075/48f9592b-2f1f-48a2-a13f-9d0a6596e90c)

**Comment by andresdelcampo** [comment]
See if the comments in this issue help -it was the same error: https://github.com/andresdelcampo/GameList_Editor/issues/8

**Comment by meatee** [comment]
Okay! I got it working. What I did was run the command mentioned here:
https://stackoverflow.com/questions/46519511/getting-eolesyserror-class-not-registered-but-dont-know-which-dll-is-missing

From an administrator PowerShell, I ran this command:
Get-WindowsCapability -online | Where-Object -Property name -like "*media*" | Add-WindowsCapability -Online

This appears to have reinstalled media components and re-registered dlls. What's weird is that I had no issues with videos before, and this is a relatively fresh install of Windows 10 Pro. I'm in the US, so this isn't an N version.
